apps/src/applab/applab.js

```diff
@@ -314,6 +314,19 @@ Applab.getCode = function() {
   return studioApp().getCode();
 };
 
+/**
+ * Helper function for levelbuilders to get the level html to copy into a widget
+ * mode level.
+ */
+Applab.getHtmlForWidgetMode = function() {
```

Comment:
it'd be useful to have a unit test for this, even just to document what the expected input/output looks like